src/typings.d.ts

```diff
@@ -3,3 +3,8 @@ declare var module: NodeModule;
 interface NodeModule {
   id: string;
 }
+
+declare module '*.json' {
+  const value: any;
+  export default value;
```

Comment:
What is this module good for?

Reply:
It's for typescript to read json files and split them into correct values.